app/views/service_requests/review/_calendars.html.haml

```diff
@@ -29,14 +29,15 @@
           arm: arm, service_request: service_request, sub_service_request: sub_service_request,
           page: pages[arm.id], pages: pages, portal: portal, admin: admin,
           review: review, merged: merged, consolidated: consolidated, statuses_hidden: %w(first_draft)
-  .otf-calendars
-    .page-header
-      %h3
-        = t(:proper)[:review][:otf_calendar_notice]
-    = render 'service_calendars/master_calendar/otf/otf_calendar', tab: tab,
-      service_request: service_request, sub_service_request: sub_service_request,
-      portal: portal, admin: admin, review: review, merged: merged,
-      consolidated: consolidated, protocol: service_request.protocol, statuses_hidden: %w(first_draft)
+  - if service_request.has_one_time_fee_services?
```

Comment:
@jayhardee9 
I'm not sure that's possible by the review step. Don't we update any SSRs to draft status in step 1?